pages/common/dvtm.md

```diff
@@ -0,0 +1,32 @@
+# dvtm
+
+> Dynamic virtual terminal manager.
+
+- Show manual page:
```

Comment:
Please show a basic invocation of dvtm. And then move to these commands.

Reply:
In most cases it exec through abduco. You don't need to invocate it manually. And since multiple consecutive examples are forbidden, what should I choose from this:
`dvtm`
`abduco -A {{name}}`
`abduco -A {{name}} dvtm`

Reply:
Then please add a sub-heading saying - Refer to the abduco command to see how to invoke this.